template/install.pkgs.owt.m4

```diff
@@ -1,9 +1,9 @@
 ifelse(BUILD_DEV,enabled,,COPY --from=build /home/owt-server/dist /home/owt)
 COPY --from=build /home/build /
 
-ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/ifelse(index(DOCKER_IMAGE,ubuntu),-1,lib64,lib/x86_64-linux-gnu)
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/ifelse(index(DOCKER_IMAGE,ubuntu),-1,lib64,lib/x86_64-linux-gnu):ifelse(index(DOCKER_IMAGE,xeon-),-1,/opt/intel/mediasdk/lib64,,)
```

Comment:
pls. fix the warning reported related to “lib64,,” -  Warning: excess arguments to builtin `ifelse' ignored